### PR TITLE
Change Query Assist sample template to use Claude model

### PR DESCRIPTION
**Pull request by dbwiddis**
### Description

Changes the sample Query Assist template to use the Claude model and adds guardrails fields.  

This matches our latest query assist template used for testing.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: sample-templates/query-assist-agent-claude.json [hunk 1]
```diff
@@ -1,90 +1,108 @@
...
+            "create_claude_connector": "connector_id"
           },
           "user_inputs": {
-            "name": "openAI-gpt-3.5-turbo",
-            "deploy": true
+            "description": "Claude model",
+            "deploy": true,
+            "name": "claude-instant",
+            "guardrails": {
+              "type": "",
```
Comment (owaiskazi19) [inline sample-templates/query-assist-agent-claude.json:62]: No default type here?

Comment (dbwiddis) [reply sample-templates/query-assist-agent-claude.json:62]: What do you want me to put there? Any non-null string works.

File: sample-templates/query-assist-agent-claude.json [hunk 2]
```diff
@@ -1,90 +1,108 @@
...
                 "method": "POST",
-                "url": "https://${parameters.endpoint}/v1/chat/completions"
+                "request_body": "{\"prompt\":\"${parameters.prompt}\", \"max_tokens_to_sample\":${parameters.max_tokens_to_sample}, \"temperature\":${parameters.temperature},  \"anthropic_version\":\"${parameters.anthropic_version}\" }",
+                "action_type": "predict",
+                "url": "https://bedrock-runtime.us-west-2.amazonaws.com/model/anthropic.claude-instant-v1/invoke"
               }
-            ]
+            ],
+            "credential": {
+                "access_key": "<put your access key here>",
```
Comment (amitgalitz) [inline sample-templates/query-assist-agent-claude.json:35]: we should need session_token also

File: sample-templates/query-assist-agent-claude.json (hunk 1 above)
Comment (dbwiddis) [reply sample-templates/query-assist-agent-claude.json]: Changed type to `local_regex` per the documentation pr

File: sample-templates/query-assist-agent-claude.json (hunk 2 above)
Comment (dbwiddis) [reply sample-templates/query-assist-agent-claude.json:35]: point me to the docs for that?

Comment (amitgalitz) [reply sample-templates/query-assist-agent-claude.json:35]: https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/bedrock_connector_anthropic_claude_blueprint.md#2-create-connector-for-amazon-bedrock

**Review by owaiskazi19 — APPROVED** [review]

File: sample-templates/query-assist-agent-claude.json (hunk 2 above)
Comment (dbwiddis) [reply sample-templates/query-assist-agent-claude.json]: done

**Review by amitgalitz — APPROVED** [review]

**Pull request merged**